libs/cart/src/drivers/interfaces/cart-service.interface.ts

```diff
@@ -11,7 +10,12 @@ export interface DaffCartServiceInterface<T extends DaffCart> {
 	/**
 	 * Retrieve a cart
 	 */
-	get(id: T['id']): Observable<T>;
+  get(id: T['id']): Observable<T>;
+
+  /**
+   * Creates a cart.
+   */
+  create(): Observable<Partial<T>>
```

Comment:
I think we should probably pull back this interface to something more minimal. 

```
{
 id: T['id']
}
```

as that is likely the common case.

Reply:
Changed in recent push.